src/routes/+layout.svelte

```diff
@@ -1,5 +1,31 @@
 <script>
   import '../app.css';
+  import Navbar from '$lib/layout/Navbar.svelte';
+  import Footer from '$lib/layout/Footer.svelte';
+  import Sidebar from '$lib/layout/Sidebar.svelte';
 </script>
 
-<slot />
+<!-- Uncomment <Sidebar /> and comment main + navbar + footer to correctly see the sidebar  -->
+<Sidebar />
+<Navbar />
+<main class="flex-grow bg-ni-primary -z-10">
+  <img
+    alt="NIAEFEUP hexagon logo"
+    class="fixed h-full sm:pb-32 overflow-hidden -right-24 sm:right-0 pt-20 pb-40"
+    src="/images/outline_white.png"
+  />
```

Comment:
Since we're using this in layout and sidebar, I'm thinking if it's worth it to have a component with a prop telling if it's placed on the right or on the left

Reply:
This has been moved to its own component with the described prop.